index.html

```diff
@@ -0,0 +1,196 @@
+                    <div class="icon-container2">
+                        <img src="./images/responsive.svg" alt="icon phone" class="icon">
+                    </div>
+                        <h3>FULLY RESPONSIVE</h3>
+                    <p class="text text-align">
+                        Looks amazing on any device:<br>
+                        smartphone, tablet, laptop and<br>
+                        desktop.
+                    </p>
+                </div>
+                <div class="items-features">
+                    <div class="icon-container2">
+                        <img src="./images/customizable.svg" alt="icon tools" class="icon">
+                    </div>
+                        <h3>CUSTOMIZABLE</h3>
+                    <p class="text text-align">
+                        Change the colors, pictures or <br>
+                    any od the sections to suit your<br>
+                    needs.
+                    </p>
+                </div>
+                <div class="items-features">
+                    <div class="icon-container2">
+                        <img src="./images/design.svg" alt="award" >
+                    </div>
+                        <h3>
+                        SLICK AND BEAUTIFUL<br>
+                        DESIGN
+                    </h3>
+                    <p class="text text-align">
+                        Trendy and fresh design, fits<br>
+                        any website.
+                    </p>
+                </div>
+            </section>
+        </div>
+    </div>
+    <div class="container section-3">
+        <div class="content-wrapper">
+             <img src="./images/logos.png" alt="img logos">
+        </div>
+    </div>
+    <div class="container section-4">
+        <div class="content-wrapper">
+                <section class="container4-section1">
+                    <img src="./images/imac.png" alt="icon imac" class="icon">
+                </section>
+            <section class="container4-section2">
+                <h2>SIMPLE AND BEAUTIFUL</h2>
+                <p>
+                    Use Nova theme for your next web project. It is<br>
+                    completely customizable so you can change any of<br>
+                    the section to fit your needs. Nova Theme is Free for<br>
+                    any kind od use, personal and commercial. Have fu<br>
+                    and good luck!
+                </p>
+            </section>
+        </div>
+    </div>
+    <div class="container section-5">
+        <div class="content-wrapper">
+            <header class="container5-header">
+                <h3>CHOOSE YOUR PRICING PLAN</h3>
+                <p class="text">Pick any of our super affordable pricing plans</p>
+            </header>
+            <section>
+                <div class="container5-section2">
+                    <div class="container5-section2--items">
+                        <h5>BASIC</h5>
```

Comment:
Chyba bym dał `h4`, bo dla sekcji nasz `h3` - więc aby było po kolei (to też nie błąd, ale jakoś tak zasadniej)